File: Go/include/terminal.go
package include

import (
	"fmt"
	"os"
)

func GotoXY(x, y int) {
	fmt.Printf("\033[%d;%dH", y, x)
	os.Stdout.Sync()
}

func HideCursor() {
	fmt.Printf("\033[?25l")
	os.Stdout.Sync()
}

func ShowCursor() {
	fmt.Printf("\033[?25h")
	os.Stdout.Sync()
}

func ClearDisplay() {
	fmt.Printf("\033[2J\033[H")
	os.Stdout.Sync()
}

File: CSharp/include/Terminal.cs
namespace Ouroboros.Include;

public static class Terminal
{
    public static void GotoXY(int x, int y) => Console.SetCursorPosition(x, y);

    public static void HideCursor() => Console.CursorVisible = false;

    public static void ShowCursor() => Console.CursorVisible = true;

    public static void ClearDisplay()
    {
        Console.Clear();
    }
}